qemu/tests/block_stream_drop_backingfile.py

```diff
@@ -98,15 +98,17 @@ def get_openingfiles():
         if set([snapshots[0], image_file]).issubset(get_openingfiles()):
             test.fail("%s is opening by qemu"
                       % set([snapshots[0], image_file]))
-        error_context.context("Check backing-file of sn2 by qemu-img", logging.info)
+        error_context.context("Reboot VM to check it works fine", logging.info)
```

Comment:
Line 88, the same issue also exist. 
        error_context.context("Check sn1 is not opening by qemu process",
                             logging.info)
         if snapshots[0] in get_openingfiles():
            test.fail("sn1 (%s) is opening by qemu" % snapshots[0])

For blockdev, delete node that not in snapshot chain after block stream/ block commit will be better.
Like:
  1. create snapshot chain: base->sn1->sn2
  2. do block stream from sn1 to sn2
  3. check snapshot chain after stream: base->sn2
      (sn1 has been removed from snapshot chain, we can remove it by blockdev-delete)
  4. delete sn1: {"execute":"blockdev-delete","arguments":{"node-name":"sn1"}}
  Suggest to add step4 in def block_stream(that supported for blockdev)

And after blockdev-delete, the logic in this file don't need to be updated any more.

Reply:
good  suggestion, thanks.

Reply:
@aliang123  will be fix in next blockdev patch, thanks.